### PR TITLE
Refactor submodules require

### PR DESCRIPTION
**Pull request by SemenchenkoVitaliy**
Refs: https://github.com/metarhia/metatests/issues/74
In order to be able to run metatests in browser, webpack should be able to parse files and preload modules. More about it here: https://github.com/metarhia/metatests/pull/75#issuecomment-416203241

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const submodules = [
   'events', // Events and emitter
   'units', // Units conversion
   'network', // Network utilities
-  'id', // Kyes and identifiers
+  'id', // Keys and identifiers
```
Comment (belochub) [inline common.js]: This should probably be done in a separate PR.

Comment (belochub) [reply common.js]: No, just open another PR with this change and remove the commit with this change from this PR.

Comment (belochub) [reply common.js]: @SemenchenkoVitaliy also there are some more typos in this file, you can fix them as well in this new PR.

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/common/commit/551bca1f7a204a3758d52da228bce032a7128131.

**Comment by SemenchenkoVitaliy** [comment]
@tshemsedinov Can you, please, release new version? We need it in order to use cli in metatests: https://github.com/metarhia/metatests/pull/75
